framework/src/modules/chain/helpers/config.js

```diff
@@ -119,7 +119,8 @@ function Config(packageJson, parseCommandLineOptions = true) {
 		},
 		redis: {
 			host: program.redis || getenv(process.env.LISK_REDIS_HOST),
-			port: parseInt(getenv(process.env.LISK_REDIS_PORT)) || null,
+			port:
+				+program.port || parseInt(getenv(process.env.LISK_REDIS_PORT)) || null,
```

Comment:
why do we need null here ? Maybe we can get rid of it?

Reply:
Redis is optional, if it's `null` it uses the default port `6380`

Reply:
Just tested this, if you remove the `|| null`, schema validation fails.
The property needs to be null so it can be replaced by default value and pass validation.

Reply:
`program.port` isn't meant to be Redis port. Is it?